namer/namer.cc

```diff
@@ -334,6 +334,13 @@ class NameInserter {
                 }
             }
         }
+        if (!klass->singletonAncestors.empty()) {
```

Comment:
We use `KeepForIDE` to support loc queries on `include`, but forgot to do the same for `extend`.